Bio/SeqIO/QualityIO.py

```diff
@@ -929,8 +929,7 @@ def FastqGeneralIterator(source: _TextIOSource) -> Iterator[Tuple[str, str, str]
     would prevent the above problem with the "@" character.
     """
     with as_handle(source) as handle:
-        if handle.read(0) != "":
-            raise StreamModeError("Fastq files must be opened in text mode") from None
+        File.check_handle_mode(handle, "r", fmt="Fastq")
```

Comment:
Style wise, I'd do ``from Bio.File import check_handle_mode`` like we do with ``as_handle`` (most consistent).

Reply:
Same for the other files (`Bio/Align/interfaces.py`, `Bio/SeqIO/Interfaces.py`, `Bio/Sequencing/Phd.py`) which currently use `File.` style?

Reply:
More changes either way :shrug:

PEP8 doesn't rule on this does it? My personal preference with just a couple of functions is to import just them (if ``from Bio.File import ...``), but it looks like as a group we've not settled on one way or the other.